server/chart_config.json

```diff
@@ -398,41 +424,82 @@
         ],
         "source": "DEA",
         "url": "https://www.deadiversion.usdoj.gov"
-      },
-      {
-        "title": "COVID-19 Cumulative Cases",
-        "chartType": "LINE",
-        "statsVars": [
-          "CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase",
-          "CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased"
-        ],
-        "source": "The New York Times",
-        "url": "https://www.nytimes.com/article/coronavirus-county-data-us.html"
       }
     ],
     "children": [
       {
-        "label": "Mortality",
+        "label": "COVID-19",
         "charts": [
           {
-            "title": "Mortality Trend",
-            "topic": "Health",
+            "title": "COVID-19 Cumulative Cases",
             "chartType": "LINE",
             "statsVars": [
-              "Count_Death_DiseasesOfTheCirculatorySystem",
-              "Count_Death_Neoplasms",
-              "Count_Death_DiseasesOfTheRespiratorySystem",
-              "Count_Death_ExternalCauses",
-              "Count_Death_DiseasesOfTheNervousSystem"
+              "CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedOrProbableCase"
             ],
-            "source": "CDC",
-            "url": "https://www.cdc.gov/"
+            "source": "The New York Times",
+            "url": "https://www.nytimes.com/article/coronavirus-county-data-us.html"
```

Comment:
maybe in later PR. the source domain is in the cache now, can use that instead of having them in config here.

Reply:
yea, i wanted to start on that soon.